From 56040609d5ab9bc071a6aac1b09e4f3b80e26bfb Mon Sep 17 00:00:00 2001
From: "Zhou, ShimingX" <shimingx.zhou@intel.com>
Date: Wed, 14 Jan 2015 14:44:30 +0800
Subject: [PATCH 397/429] systemd: Enable network namespacing feature in kernel

This patch fixed systemd-hostnamed failed issue when booting
on step network.

Signed-off-by: zhoushix <shimingx.zhou@intel.com>
---
 arch/x86/configs/i386_edison_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/x86/configs/i386_edison_defconfig b/arch/x86/configs/i386_edison_defconfig
index d8da11b9b4a7..b1ed708a94cb 100644
--- a/arch/x86/configs/i386_edison_defconfig
+++ b/arch/x86/configs/i386_edison_defconfig
@@ -153,7 +153,7 @@ CONFIG_NAMESPACES=y
 # CONFIG_IPC_NS is not set
 # CONFIG_USER_NS is not set
 # CONFIG_PID_NS is not set
-# CONFIG_NET_NS is not set
+CONFIG_NET_NS=y
 CONFIG_UIDGID_CONVERTED=y
 # CONFIG_UIDGID_STRICT_TYPE_CHECKS is not set
 CONFIG_SCHED_AUTOGROUP=y
-- 
2.37.3

